### PR TITLE
feat: add current PR status to ftrack-sync action

### PR DESCRIPTION
**Pull request by gismya**
Test-ID: FT-af5d1e42-c544-11eb-97da-4e4b284cd95b

- [x] I have added automatic tests where applicable
- [x] The PR title is suitable as a release note
- [x] The PR contains a description of what has been changed
- [x] The description contains manual test instructions

## Changes

Added a row in the comment synced to ftrack with the current PR status, e.g draft, merged, open.
Updated old tests and created new tests to test the new changes.

## Test

Run the test suite. Keep an eye out in the ftrack notes to ensure all the edge cases actually get the correct status.

### CODE REVIEW

File: actions/ftrack-sync/ftrackSync.js [hunk 1]
```diff
@@ -56,12 +56,20 @@ async function groupIntoExistingAndNewNoteIds(noteIds) {
...
+  const prUrl = pr.html_url;
   const linkDescription = prUrl.match(/\.com\/(.+)/)[1];
-
+  const prStatus = pr.draft
+    ? "draft"
+    : !!pr.merged_at
+    ? "merged"
+    : !!pr.state
+    ? pr.state
+    : "unknown";
```
Comment (jimmycallin) [inline actions/ftrack-sync/ftrackSync.js:68]: please make it as a regular if statement instead :)

**Review by jimmycallin — CHANGES_REQUESTED** [review]
image
please fix newline issue as well

**Review by jimmycallin — APPROVED** [review]
